third-party/snyk/snapshot.json

```diff
@@ -57,6 +58,8 @@
       {"id":"SNYK-JS-GOOGLECLOSURELIBRARY-174519","severity":"medium","semver":{"vulnerable":[">=20190121.0.0 <20190301.0.0"]}}
     ],
     "handlebars":[
+      {"id":"SNYK-JS-HANDLEBARS-534988","severity":"high","semver":{"vulnerable":["<4.5.3"]}},
+      {"id":"SNYK-JS-HANDLEBARS-534478","severity":"high","semver":{"vulnerable":["<4.5.3"]}},
```

Comment:
these are in fact two separate vulns 

https://snyk.io/vuln/SNYK-JS-HANDLEBARS-534988
https://snyk.io/vuln/SNYK-JS-HANDLEBARS-534478